app/views/bots/index.html.erb

```diff
@@ -16,7 +16,7 @@
             <% if bot.description.present? %><p><%= bot.description %></p><% end %>
             <% if bot.repository.present? %><p class="text-muted">Repository: <%= repo_link(bot) %></p><% end %>
             <p><strong><%= link_to "Instances", url_for(controller: :bot_instances, action: :index, bot_id: bot.id) %></strong></p>
-            <ul class="bot-instance-ul">
+            <ul class="bot-instance-ul<%= " no-collaborators" unless User.with_role(:collaborator, bot).any? %>">
```

Comment:
There's a helper method you can use instead of the `with_role` call - I think it's `Bot#collaborators`, but you might want to make sure. It'll be defined in the model.